### DISCUSSION TITLE
Matrix from getMatrixFromProject

### DISCUSSION THREAD

**Discussion by AnjaliC4** [GreenleafLab/ArchR#1784] [Questions / Documentation]
Hi @rcorces,

I was wondering whether both the GeneIntegrationMatrix and GeneScoreMatrix exported from ArchR using getMatrixFromProject() are raw or log2 normalized counts?

In addition, is there a way to export MAGIC imputed matrices?

Thanks very much!

**Comment by AnjaliC4** [comment]
Hi @rcorces, would really appreciate your input on this whenever you have a chance. 

Thank you

**Comment by rcorces** [comment]
Sorry for the delay. Its not so straightforward. The GeneIntegrationMatrix undergoes normalization (via `Seurat::NormalizeData()`) and scaling (via `Seurat::ScaleData()`). GeneScoreMatrix the exact math depends on the parameters you pass to the function. But the output is neither raw or log2 normalized counts per se and typically there are weights applied etc.

regarding exporting imputed matrices - no that is not possible because those matrices arent stored. They are calculated on the fly via the `imputeMatrix()` function. You could, of course, use that function yourself to obtain them.
